Common/Core/vtkCommand.h

```diff
@@ -243,10 +243,13 @@
     _vtk_add_event(ExitEvent)\
     _vtk_add_event(LeftButtonPressEvent)\
     _vtk_add_event(LeftButtonReleaseEvent)\
+    _vtk_add_event(LeftButtonDoubleEvent)\
```

Comment:
We need to include "Click" in the name to make its meaning more clear: LeftButtonDoubleClickEvent. Same for  middle and right.